src/node/audio_buffer_source.rs

```diff
@@ -327,34 +286,44 @@ impl AudioBufferSourceNode {
         &self.detune
     }
 
+    pub fn loop_state(&self) -> &LoopState {
```

Comment:
This is not in the spec and exposes an internal implementation detail, probably we should stick to the "Keep Your API Surface Small" principle here